iree/compiler/Dialect/HAL/Target/VulkanSPIRV/VulkanSPIRVTarget.cpp

```diff
@@ -108,6 +109,108 @@ static void recordFullExecutionBarrier(Value commandBuffer, Location loc,
       ArrayRef<Value>{memoryBarrier}, ArrayRef<Value>{});
 }
 
+/// Generates IR to compute the ceil(`numerator`, `denominator`).
+static Value computeCeil(Location loc, Value one, Value numerator,
```

Comment:
I think this is called "ceilDiv" instead of just "ceil"?

Reply:
Right! Thanks!